gf-button/README.md

```diff
@@ -34,81 +33,78 @@ GFButton(
 ),
 ```
 
-GF Buttons can be styled with several attributes to look a specific way. Each and every attribute is described below.
+## Propiedades
 
-## Properties 
+Se pueden utilizar muchos atributos para darle estilo a los botones GFButton y que luzcan de una manera espec&iacute;fica. Cada uno de los atributos se describe a continuaci&oacute;n.
 
-GF Buttons can be styled with several attributes to look a specific way. Each and every attribute is described below.
+### Color
 
-### Color 
-
-GFColor is used to change the background of the button.
+GFColor es utilizado para cambiar el color de fondo del bot&oacute;n.
 
 |  |  |
 | :--- | :--- |
-| **Description** | The color to use from application's color palette                                |
-| **Attribute** |  color |
-| **Type** | `GFColor` |
-| **Default** | `GFColor.primary` |
+| **Descripci&oacute;n** | El color a utilizar de la paleta de colores de la aplicaci&oacute;n |
+| **Atributo** |  color |
+| **Tipo** | `GFColors` |
+| **Default** | `GFColors.PRIMARY` |
 
-### Size
+### Tama&ntilde;o
 
 |  |  |
 | :--- | :--- |
-| **Description**  | The GFButton Size                                                                                           |
-| **Attribute** |  size |
-| **Type** | `GFSize.large,  GFSize.medium, GfSize.small` |
-| **Default** | `GFSize.medium` |
+| **Descripci&oacute;n** | El tama&ntilde;o del bot&oacute;n |
+| **Atributo** | size |
+| **Tipo** | `GFSize.LARGE, GFSize.MEDIUM, GFSize.SMALL` |
+| **Default** | `GFSize.MEDIUM` |
 
-### Shape
+### Forma
 
 |  |  |
 | :--- | :--- |
-| **Description**     |  The GFButton Shape                                               |
-| **Attribute** |  shape |
-| **Type**               | `GFButtonShape.standard, GFButtonShape.square, GFButtonShape.pills` |
+| **Descripci&oacute;n** | La forma del bot&oacute;n |
+| **Atributo** | shape |
+| **Tipo** | `GFButtonShape.standard, GFButtonShape.square, GFButtonShape.pills` |
 | **Default** | `GFButtonShape.standard` |
 
-### Type
+### Tipo
 
 |  |  |
 | :--- | :--- |
-| **Description**     |   The GFButton Type                                |
-| **Attribute** |   type |
-| **Type** | `GFType.solid, GFType.outline, GFType.outline2x, GFType.transparent` |
-| **Default** | `GFType.solid` |
+| **Descripci&oacute;n** | El tipo del bot&oacute;n |
+| **Atributo** | type |
+| **Tipo** | `GFButtonType.solid, GFButtonType.outline, GFButtonType.outline2x, GFButtonType.transparent` |
+| **Default** | `GFButtonType.solid` |
 
-### Position
+### Posici&oacute;n
 
 |  |  |
 | :--- | :--- |
-| **Description** |  The GFButton Position                                                                    |
```

Comment:
It says that is the position of the button, but it isn't, instead, it is the position of the icon in the button.